PyInstaller/utils/conftest.py

```diff
@@ -439,13 +424,19 @@ def _test_building(self, args):
 
         Return True if build succeded False otherwise.
         """
-        default_args = ['--debug=bootloader', '--noupx',
-                '--specpath', self._specdir,
-                '--distpath', self._distdir,
-                '--workpath', self._builddir,
-                '--path', _get_modules_dir(self._request),
-                        '--log-level=INFO',
-                ]
+        default_args = [
+            '--debug=bootloader',
+            '--noupx',
+            '--specpath',
+            self._specdir,
+            '--distpath',
+            self._distdir,
+            '--workpath',
+            self._builddir,
+            '--path',
+            _get_modules_dir(self._request),
+            '--log-level=INFO',
+        ]
```

Comment:
It might be nicer to have command-line switch and its value on the same line. Though perhaps the --debug=bootloader and --log-level=INFO entries should be split in two for the sake of consistency...